ci/deps/travis-37.yaml

```diff
@@ -22,3 +22,4 @@ dependencies:
   - pip
   - pip:
     - moto
+    - pyreadstat
```

Comment:
is this only a wheel?

Reply:
add to one of the windows builds & the macosx build; is this support on 3.5? any other requirements?

Reply:
Done.

Reply:
I don't know what you mean by "only a wheel" though. `pyreadstat` has binary wheels for Windows, Linux, and macOS for Python 3.5, 3.6, and 3.7: https://pypi.org/project/pyreadstat/#files

Reply:
meaning you are installing thru pip, rather use a conda package if its available